iridauploader/parsers/directory/sample_parser.py

```diff
@@ -221,7 +221,7 @@ def _parse_samples(sample_sheet_file):
             value = line[index].strip()
 
             # Keys other than 'File_Reverse' cannot be empty
-            if len(value) is 0:  # no value
+            if len(value) == 0:  # no value
```

Comment:
iirc, the `is` operator should only be used for comparing objects/booleans. `==` is the correct operator for integer/string comparison. When it runs with `is` it transforms the return from `len(value)` as a boolean (0 when 0, and 1 when >0), and compares it to the boolean 0. If it runs with `==` then it just compares integers like intended. Functionally this has difference since we are comparing against 0, but in the future if someone would modify the function without knowing this difference a bug could be introduced.

The latest pep8 tests showed this as a warning so I just went around and fixed them.